doc/guide.md

```diff
@@ -2816,6 +2816,9 @@ leave it unmodified (default).
   Defaults to 10000.
 * `token_cache_ttl_ms`: Time to live in milliseconds for token cache entries.
   Defaults to 180000 (3 minutes).
+* `deploy.waitIterations`: Maximum number of iterations for deployment - before
+  giving up (readiness check). A value, `n`, corresponds to roughly  `n*n*0.2` seconds.
+  This value, if set, overrides the `waitIterations` in the launch descriptor.
```

Comment:
Can you extend
https://github.com/folio-org/okapi/blob/master/dist/okapi.conf
https://github.com/folio-org/okapi/blob/master/dist/okapi.sh
accordingly?

Reply:
The okapi.conf and okapi.sh are very problematic. Any time we need to add *anything* they have to be updated. Had they just made an option to refer to user-defined okapi.conf (with Vert.x properties) once and for all, NOTHING would have to be updated.

It's probably legacy from times when okapi.conf did not include "everything".